cmds/core/ping/ping.go

```diff
@@ -158,6 +160,9 @@ func main() {
 		if err != nil {
 			log.Fatalf("ping failed: %v", err)
 		}
+		if *audible {
```

Comment:
I am wondering if you might use https://godoc.org/github.com/pborman/ansi for your bell instead of \a. 
see ansi.Bell

the other thing: we're using log.Print to print, and you are using fmt.Print. These go separate places. You might do this:
if *audible {
msg = ansi.Bell + msg
}

then you don't have the separate print